pkg/apis/apps/install/install.go

```diff
@@ -0,0 +1,125 @@
+/*
```

Comment:
Yeah agree with desire to create its own group, I definitely want a
different name before we merge (will think hard on this).

On Sat, Apr 16, 2016 at 12:50 AM, Prashanth B <notifications@github.com>
wrote:

> In pkg/apis/apps/install/install.go
> <https://github.com/kubernetes/kubernetes/pull/24315#discussion_r59962497>
> :
>
> > @@ -0,0 +1,125 @@
> > +/*
>
> I think the idea is to get daemon,replica etc out of "extensions". Open to
> suggestion, apps is just something we came up with after ~10m of
> discussions, and no one really had anything better.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/24315/files/5fa8ac534e922d68815c7979c25304b714d6c0c5..8d57f836a17d3eabd7c37583a61f395e2018a902#r59962497>
>